Documentation/UserInterfaceRecipes.md

```diff
@@ -0,0 +1,128 @@
+# User Interface Recipes
+
+### ViewModel / Presenter / ViewData Objects
+
+Regardless of what you call them, or how much responsibility they should (or should not) have in your project, generally these objects contain a 1:1 mapping of properties to the controls on your screen. The type of properties you use (`MutableProperty`, `ConstantProperty`, or `AnyProperty`) depends on your specific needs.
+
+For example, consider a UI that displays both a slider and a text field that represent a magnitude. The text field specifies its values in dBFS (decibels, full scale). A common approach is to expose these values in your ViewModel using only a single `MutableProperty`:
+
+``` swift
+final class VolumeViewModel {
```

Comment:
I'm not keen at all on the word "view model". The term is overloaded and it has too much baggage.
To avoid confusion, how about just `VolumeData`, or if you want, `VolumeViewData`?